File: fibonacci/rust/code.rs
fn main() {
    let u: i32 = std::env::args()
        .nth(1)
        .unwrap()
        .parse()
        .unwrap();
    let mut r = 0;
    for i in 1..u {
        r += fibonacci(i);
    }
    println!("{}", r);
}

fn fibonacci(n: i32) -> i32 {
    match n {
        0 => 0,
        1 => 1,
        _ => fibonacci(n - 1) + fibonacci(n - 2),
    }
}

File: fibonacci/dart/code.dart
int fibonacci(int n) {
  if (n == 0) return 0;
  if (n == 1) return 1;
  return fibonacci(n - 1) + fibonacci(n - 2);
}

void main(List<String> args) {
  final u = int.parse(args[0]);
  int r = 0;

  for (int i = 0; i < u; i++) {
    r += fibonacci(i);
  }
  print(r);
}
